frontend/src/components/design-system/buttons/IconButton.vue

```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-export enum ButtonSizeEnum {
+export enum IconButtonSizeEnum {
```

Comment:
Tohle jeste zmenim v jinym PR aby to sedelos logikou `size: 'small'|'medium'|'large'` misto enumu